Explorer/Assets/DCL/MapRenderer/MapLayers/UsersMarker/HotUserMarker.cs

```diff
@@ -25,6 +27,8 @@ internal HotUserMarker(IObjectPool<HotUserMarkerObject> pool, ICoordsUtils coord
             poolableBehavior = new MapMarkerPoolableBehavior<HotUserMarkerObject>(pool);
         }
 
+        public void ToggleSelection(bool isSelected) { }
+
         public void UpdateMarkerPosition(string playerId, Vector3 position)
         {
             CurrentPlayerId = playerId;
```

Comment:
`CurrentPlayerId` seems like it can be deleted. Its not being used outside the interface